CMakeLists.txt

```diff
@@ -128,6 +128,8 @@ endif()
 add_definitions(
   -DOSQUERY_BUILD_VERSION=${OSQUERY_BUILD_VERSION}
   -DOSQUERY_BUILD_SDK_VERSION=${OSQUERY_BUILD_SDK_VERSION}
+  -DOSQUERY_BUILD_OS=${OSQUERY_BUILD_OS}
```

Comment:
Want to call this `OSQUERY_BUILD_PLATFORM` and `OSQUERY_BUILD_PLATFORM_VERSION`?